src/Extension/ClamAVExtension.php

```diff
@@ -163,15 +161,7 @@ public function getFullPath()
             return null;
         }
 
-        $fileMetaData = $this->owner->File->getMetadata();
-
-        if ($this->owner->isPublished()) {
-            return PUBLIC_PATH . $this->owner->File->getURL();
-        } else {
-            return ASSETS_PATH . '/' .
-                Config::inst()->get(ProtectedAssetAdapter::class, 'secure_folder')
-                . '/' . $fileMetaData['path'];
-        }
+        return PUBLIC_PATH . $this->owner->File->getURL();
```

Comment:
I don't think we can remove the logic for unpublished files here.

Reply:
Yeah ... you probably want to bring that back. 

I might want to add that this doesn't seem to handle the scenario where the file has a published version and a different draft version.